make/02_mod.mk

```diff
@@ -36,6 +36,7 @@ include make/e2e-setup.mk
 include make/scan.mk
 include make/ko.mk
 include make/third_party.mk
+include make/_shared_new/helm/crds.mk
```

Comment:
Instead of kloning in `make/_shared` and auto including these targets, we kloned to `make/_shared_new/helm` and include the Makefile here manually. This is done so that we only include the crd generation (Helm chart generation is for a future PR).